src/z-form.cc

```diff
@@ -556,7 +556,7 @@ unsigned int vstrnfmt(char *buf, unsigned int max, const char *fmt, va_list vp)
  */
 static char *vformat(const char *fmt, va_list vp)
 {
-	static char *format_buf = NULL;
+	static char *format_buf = (char *)NULL;
```

Comment:
Can you check if a plain "nullptr" without the cast is accepted by your compiler here?

Reply:
Done, compiles OK, and I agreee that it's neater this way.